packages/terra-framework-docs/src/terra-dev-site/test/tabs/Tabs/AddCloseTab.test.jsx

```diff
@@ -0,0 +1,119 @@
+import React, { useState } from 'react';
+import classNames from 'classnames/bind';
+import PropTypes from 'prop-types';
+import IconAddPerson from 'terra-icon/lib/icon/IconAddPerson';
+import IconSearch from 'terra-icon/lib/icon/IconSearch';
+import Tabs from 'terra-tabs';
+import TabContentTemplate from './TabContentTemplate';
+import styles from './TabsTemplate.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  containerClassName: PropTypes.string,
+};
+
+const defaultProps = {
+  containerClassName: 'content-wrapper-default',
+};
```

Comment:
This is not required. You can directly apply the className

Reply:
Done https://github.com/cerner/terra-framework/commit/c007c2b50613c2908c4aab55530060a38ea67584